src/intersection-observer.js

```diff
@@ -81,20 +81,29 @@ export function getIntersectionChangeEntry(element, owner, viewport) {
       'Negative dimensions in element.');
   // Building an IntersectionObserverEntry.
 
-  let intersectionRect = element;
+  // The element is relative to (0, 0), while the viewport moves. So, we must
+  // adjust.
+  const boundingClientRect = moveLayoutRect(element, -viewport.left,
```

Comment:
Do you mean that you will address `position:fixed` issue in a follow up? If so, please add `TODO(#issue)`.